### DISCUSSION TITLE
Delay is not respected (without comments)

### DISCUSSION THREAD

**Discussion by dhaavi** [tiangolo/issue-manager#30] [Questions] [answered]
The issue-manager closed an issue yesterday after just 3 days, though the delay is configured as 30 days.
Something that is unusual about this case is that there was only the original post, without any additional comment.

Affected Issue: https://github.com/safing/portmaster/issues/349
Issue Manager config: https://github.com/safing/portmaster/blob/develop/.github/workflows/issue-manager.yml

**Comment by tiangolo** [comment, marked as answer]
Hello, the delay is parsed by Pydantic: https://docs.pydantic.dev/latest/concepts/types/#datetime-types

I think in Pydantic v1 it would parse integers up to some value as seconds, and above that value as milliseconds (I think as an effort to handle compatibility with JS milliseconds numbers by default).

I think that's no longer like that in Pydantic v2.

I tried checking your workflow but it seems it no longer uses issue-manager, so I suspect you handled your need already. :nerd_face:
